tests/test_gpts_coarse.py

```diff
@@ -0,0 +1,21 @@
+import gpts
+
+QUESTIONS_AND_ANSWERS = {
+    "How many ounces are there in a pound?": '16',
+    "How many eggs are in a dozen?": '12',
+    "How many days are in a week?": '7',
+}
+
+def ask_typed_questions(model):
+    for question, answer in QUESTIONS_AND_ANSWERS.items():
+        assert abs(model.ask_for(question, type=int))   == int(answer)
```

Comment:
Note: the `abs()` calls on lines 11 and 12 are a bald faced lie which we'll need to remove following Hudson's suggestions to implement dependent types like "positive integer" and "string that matches a regex."